jina/drivers/evaluate.py

```diff
@@ -20,6 +21,13 @@ def __call__(self, *args, **kwargs):
                              zip(self.req.docs, self.req.groundtruths)]
         self._traverse_apply(docs_groundtruths, *args, **kwargs)
 
+    @property
+    def id(self):
```

Comment:
```suggestion
    def metric(self):
```

I refactored this name to metric in last PR